docs/map.xsd

```diff
@@ -59,13 +131,31 @@
   </xs:restriction>
 </xs:simpleType>
 
-<xs:simpleType name="hexColourT">
+  <xs:simpleType name="cornerterrainT">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="[0-9]{1,3},[0-9]{1,3},[0-9]{1,3},[0-9]{1,3}"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+<xs:simpleType name="hexColorT">
   <xs:restriction base="xs:string">
-    <xs:pattern value="[a-fA-F0-9]{6}"/>
+    <xs:pattern value="([a-fA-F0-9][a-fA-F0-9]){3}"/>
   </xs:restriction>
 </xs:simpleType>
 
-<xs:complexType name="simpleDataT">
+  <xs:simpleType name="pointsT">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="(\[([0-9])+,([0-9])+\])+"/>
```

Comment:
Hmm, actually I just realized they are floating point (double) values, and they can also be negative. So this positive integer pattern is not going to cut it.

I wondered if SVG, which uses the same list of points format, had anything useful in [its XSD file](https://www.w3.org/TR/2002/WD-SVG11-20020108/SVG.xsd), but they just do:
```xml
<simpleType name="PointsType">
  <annotation>
    <documentation>a list of points</documentation>
  </annotation>
  <restriction base="string"/>
</simpleType>
```